src/app/space/forge-wizard/components/single-input/single-input.component.less

```diff
@@ -0,0 +1,30 @@
+@import (reference) '../../../../../assets/stylesheets/shared/osio.less';
+@icon-path: '../../../../../assets/stylesheets/shared/_icons.less';
+:host {
+  .field-required {
+    &:after {
+      content: '*';
+      padding: 3px;
+      color: @color-pf-red;
+      vertical-align: center;
+    }
+  }
+  &.has-danger {
+    .field-required {
+      &:after {
+        content: '*';
+        color: @color-pf-red;
+      }
+    }
+    .form-control-danger {
```

Comment:
There are quite a few nested styles in this file; BEM naming should be used and nesting removed to avoid specificity problems.